charts/darts-portal/values.stg.template.yaml

```diff
@@ -5,3 +5,4 @@ nodejs:
   environment:
     DARTS_API_URL: https://darts-api.staging.platform.hmcts.net
     DARTS_PORTAL_URL: https://darts-staging.staging.apps.hmcts.net
+    MANUAL_DELETION_ENABLED: true
```

Comment:
I think we need to remove this, we want to default it to false.

Reply:
I think we can disable this for prod but keep enable this for lower env. Please advise.

Reply:
The functional tests don't actually run against the version deployed to staging by the pipeline. Let me check where we need to inject the config for the process it runs.

Reply:
done